bin/activate

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+declare prompt
+declare -a sites
+declare -i default=1
+declare -i choice
+declare -i n_sites
+
+# List sites by looking in the "sites" directory
+# and store them in an array
+read -r -a sites <<< $(
+    find "${SITES_DIRECTORY}" -maxdepth 1 -mindepth 1  -type d |
+    sed 's|'"${SITES_DIRECTORY}\/"'||' |
+    xargs
+)
+n_sites=${#sites[@]}
+
+prompt="Select an available site to activate:\\n"
+for (( i=0; i<n_sites; i++ )); do
+    prompt+="[$((i+1))] ${sites[$i]}"
+    if [[ $((i+1)) -eq ${default} ]]; then
+        prompt+=" (default)"
+    fi
+    prompt+="\\n"
+done
+prompt+="Your choice: "
+read -r -p "$(echo -e "${prompt}")" choice
+
+if [[ ${choice} -gt n_sites ]]; then
+    (>&2 echo "Invalid choice ${choice} (should be <= ${n_sites})")
+    exit 10
+fi
+
+if [[ ${choice} -le 0 ]]; then
+    choice=${default}
+fi
+
+ACTIVE_SITE="${sites[$((choice-1))]}"
```

Comment:
Why not using the same semantic as in the `Dockerfile`, _e.g._ `SITE` instead of `ACTIVE_SITE`?

Reply:
Because this is exported so I thought the name should be different and more explicit.

Reply:
`RICHIE_SITE` would be more explicit then?

Reply:
ok